smtp.go

```diff
@@ -0,0 +1,165 @@
+package emaildriver
+
+import (
+	"bytes"
+	"crypto/tls"
+	"errors"
+	"net"
+	"net/smtp"
+	"strconv"
+)
+
+type SmtpEmailBackend struct {
+	isEnabled  bool
+	smtpAuth   smtp.Auth
+	serverName string
+	conn       net.Conn
+}
+
+func (m *SmtpEmailBackend) Init(conf map[string]string) error {
+	var ok bool
+	var tlsConn bool
+	var login string
+	var password string
+	var port string
+	var err error
+
+	if conf == nil {
+		return errors.New("SmtpEmailBackend requires a configuration map")
+	}
+
+	m.serverName, ok = conf["servername"]
+	if !ok {
+		return errors.New("No servername defined, emails will fail")
+	}
+
+	login, ok = conf["login"]
+	if !ok {
+		return errors.New("No login defined, emails will fail")
+	}
+
+	password, ok = conf["password"]
+	if !ok {
+		return errors.New("No password defined, emails will fail")
+	}
+
+	port, ok = conf["port"]
+	if !ok {
+		port = "25"
+	}
+
+	tlsConnTxt, ok := conf["tls"]
+	if !ok {
+		tlsConn = false
+	} else {
+		tlsConn, _ = strconv.ParseBool(tlsConnTxt)
+	}
+
+	m.smtpAuth = smtp.PlainAuth("", login, password, m.serverName)
+
+	if tlsConn {
+		tlsconfig := &tls.Config{
+			InsecureSkipVerify: true,
+			ServerName:         m.serverName,
+		}
+		m.conn, err = tls.Dial("tcp", m.serverName+":"+port, tlsconfig)
+	} else {
+		m.conn, err = tls.Dial("tcp", m.serverName+":"+port, nil)
```

Comment:
This bit of code will run when `tlsConn == false`. 

As such, it feels like we should be using `tls.Dial("tcp", m.serverName+":"+port)` rather than `tls.Dial...` function. Otherwise, I think it will try to initiate a tls handshake.